lib/floki/raw_html.ex

```diff
@@ -18,35 +18,50 @@ defmodule Floki.RawHTML do
     "wbr"
   ]
 
-  def raw_html(html_tree), do: raw_html(html_tree, "")
-  defp raw_html([], html), do: html
-  defp raw_html(string, _html) when is_binary(string), do: HtmlEntities.encode(string)
-  defp raw_html(tuple, html) when is_tuple(tuple), do: raw_html([tuple], html)
+  @doc """
+  """
+  def raw_html(html_tree, options) do
+    encoder =
+      case Keyword.fetch(options, :encode) do
+        {:ok, true} -> &HtmlEntities.encode/1
+        {:ok, false} -> &(&1)
+        :error -> &HtmlEntities.encode/1
```

Comment:
thoughts on making the default configurable at the Application level?

Reply:
This would be a nice to have feature too. It could fetch from `Application.get_env(:floki, :encode_raw_html)` on the `:error` clause of this `case`. Would be nice if we document that option too.